docs/en/wallet-attestation.rst

```diff
@@ -30,271 +33,271 @@ The following requirements for the Wallet Attestation are met:
+
+**Step 8**: The Device Integrity Service:
+
+  * generate a Key Attestation holder binded with ``challenge`` and Wallet Hardware public key.
+  * adds relevant information on the security of the device.
+  * sign the Key Attestation with an OEM certificate which in addition to certifying the integrity of the Wallet Instance also certifies that the Wallet Hardware Keys is secure stored by the OS (hardware backed).
+
+**Step 9**: The Wallet Instance send the ``challenge`` with Key Attestation and Wallet Hardware Key Tag to the Wallet Provider Backend in order to register the Wallet Instance identified by the Wallet Hardware Key public key.
+
+.. note::
+
+  The Key Attestation (``key_attestation``) must always be encoded in base64 as the format is different between the various operating systems.
 
-The response is the `Wallet Attestation`_ in JWT format:
 
 .. code-block:: http
 
-    HTTP/1.1 201 OK
-    Content-Type: application/jwt
+    PUT /wallet-instance HTTP/1.1
+    Host: walletprovider.example.com
+    Content-Type: application/json
+
+    {
+      "challenge": "0fe3cbe0-646d-44b5-8808-917dd5391bd9",
+      "key_attestation": "o2NmbXRvYXBwbGUtYXBw... redacted",
+      "hardware_key_tag": "WQhyDymFKsP95iFqpzdEDWW4l7aVna2Fn4JCeWHYtbU="
+    }
+
+.. note::
+  It is not necessary to send the Wallet Hardware public key as it is already included in the ``key_attestation``.
+
+**Steps 10-12**: The Wallet Provider validate the ``challenge`` and ``key_attestation`` signature:
+
+  1. It MUST verify that the ``challenge`` was generated by  Wallet Provider and has not already been used.
+  2. It MUST validate the ``key_attestation`` as defined by the device manufacturers' guidelines.
+  3. It MUST verify that the device in use has no security flaws and reflects the minimum security requirements defined by the Wallet Provider.
+  4. If these checks are passed, it MUST register the Wallet Instance, keeping the Wallet Hardware Key Tag and all useful information related to the device.
+  5. It SHULD associate the Wallet Instance with a specific user uniquely identified within the Wallet Provider's systems. This will be useful for the lifecycle of the Wallet Instance and for a future revocation.
```

Comment:
```suggestion
  5. It SHOULD associate the Wallet Instance with a specific user uniquely identified within the Wallet Provider's systems. This will be useful for the lifecycle of the Wallet Instance and for a future revocation.
```